### ISSUE TITLE
Exclude paths from relativize_paths filter

### ISSUE THREAD

**Issue by denisdefreyne** [nanoc/features#41]
Some paths are not meant to be relativized. For example, anything under `/cgi-bin/` should probably remain there, without a relative path. 

Add an option to exclude paths from `relativize_paths`. For example:

```ruby
  filter :relativize_paths, type: :html, exclude: %r{\A/cgi-bin/}
```

**Comment by denisdefreyne** [comment]
A work-in-progress implementation lives at nanoc/nanoc/pull/1340.

**Comment by denisdefreyne** [comment]
This will be released in 4.9.3.
